python/sglang/srt/managers/schedule_batch.py

```diff
@@ -56,6 +56,7 @@
     "disable_mla": ServerArgs.disable_mla,
     "torchao_config": ServerArgs.torchao_config,
     "disable_nan_detection": ServerArgs.disable_nan_detection,
+    "enable_dp_mla": ServerArgs.enable_dp_mla,
```

Comment:
I feel this is quite general, can we call it `enable_dp_attention`?

Reply:
Sure.